.clang-tidy

```diff
@@ -0,0 +1,2 @@
+Checks: 'bugprone-*,performance-*,modernize-*,readability-*,cppcoreguidelines-*'
```

Comment:
Highly recommended we do one-per-line.  Much easier to read visually and for Git to diff when we start tweaking checks.